### DISCUSSION TITLE
Add Android/iOS build of the app

### DISCUSSION THREAD

**Discussion by helgoboss** [helgoboss/helgobox#1016] [Ideas]
This includes at the very least:

- Making some parts of the app more suitable for touch screen
- Hiding parts that are not suitable for touch screen yet
- Setting up remote connection stuff

**Comment by chords-chords** [comment]
This app would be a remote controller for Playtime?

**Reply by helgoboss** [reply]
It would be the same interface that Playtime has right now (although tailored a bit more to touch screen usage). And it would remote-control your desktop computer, bi-directional. In the beginning over network only. In the future probably also over USB.

You could even control one Playtime instance from multiple devices.

The main drawbacks would be:

- Whenever you click on something that opens something in REAPER (MIDI editor, FX window, etc.), you will see that on the desktop computer, not on the mobile device. That's why I gradually want to add alternatives for those that are built right into the Playtime app.
- performance depends on network speed.

**Reply by chords-chords** [reply]
Thanks for the clarification. Sounds awesome!
